src/orthogonalize.jl

```diff
@@ -1,7 +1,13 @@
+using KrylovKit: schursolve, Arnoldi
 # TODO: call as `orthogonalize(ψ, -∞)`
 # TODO: could use commontags(ψ) as a default for left_tags
 function right_orthogonalize(
-  ψ::InfiniteMPS; left_tags=ts"Left", right_tags=ts"Right", tol::Real=1e-12
+  ψ::InfiniteMPS;
+  left_tags=ts"Left",
+  right_tags=ts"Right",
+  tol::Real=1e-12,
+  eager=true,
+  htol::Real=1e-10,
```

Comment:
I would call this `ishermitian_rtol`, no need for a short name here. Shouldn't we define it relative to `tol`, for example `ishermitian_rtol=10tol`?

Also if the Hermitian check is causing trouble, maybe we should just turn it into a warning.

Reply:
Or better yet, I would call this `ishermitian_kwargs` and set a default like `ishermitian_kwargs=(; rtol=10tol)`.

Reply:
Sure thing, I'll do both